crates/router/tests/connectors/adyen_uk_ui.rs

```diff
@@ -192,6 +192,35 @@ async fn should_make_adyen_twint_payment(driver: WebDriver) -> Result<(), WebDri
     Ok(())
 }
 
+async fn should_make_adyen_dana_payment(driver: WebDriver) -> Result<(), WebDriverError> {
+    let conn = AdyenSeleniumTest {};
+    conn.make_redirection_payment(
+        driver,
+        vec![
+            Event::Trigger(Trigger::Goto(&format!("{CHEKOUT_BASE_URL}/saved/175"))),
+            Event::Trigger(Trigger::Click(By::Id("card-submit-btn"))),
+            Event::Trigger(Trigger::SendKeys(
+                By::XPath("/html/body/div[1]/div[2]/div[1]/div/input"),
```

Comment:
Prefer using a unique ID or class or minimum xpath , if the website updates with new structure, this test will fail. Only go with full Xpath when u can't find any of the above.

Reply:
Done👍